Added All the Pages

stancheta opened the pull request:
Created all of the principle pages and all of them follow the structure of the
first principle page. Created all of the partial pages as well.

I made some small edits to the img names and editted some more of p1.

I have not added anything new in terms of principle partial content.

stancheta committed:
Added All the Pages

Created all of the principle pages and all of them follow the structure of the
first principle page. Created all of the partial pages as well.

I made some small edits to the img names and editted some more of p1.

I have not added anything new in terms of principle partial content.

stancheta committed:
Merge branch 'master' of github.com:redaikidoka/ebfm into steven

updating from master

redaikidoka commented:
Rock on!

On Aug 2, 2016 3:34 AM, "Steven T Ancheta" notifications@github.com wrote:

> Created all of the principle pages and all of them follow the structure of
> the
> first principle page. Created all of the partial pages as well.
> 
> I made some small edits to the img names and editted some more of p1.
> 
> ## I have not added anything new in terms of principle partial content.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/redaikidoka/ebfm/pull/3
> Commit Summary
> - Added All the Pages
> 
> File Changes
> - _R_ src/assets/img/p1-1-Cutter.jpg
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-0 (0)
> - _A_ src/assets/img/p1-1-Dyson.jpg
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-1 (0)
> - _R_ src/assets/img/p1-1-ebs-slow.gif
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-2 (0)
> - _R_ src/assets/img/p1-2-harvest.png
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-3 (0)
> - _R_ src/assets/img/p1-2-harvestfish.png
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-4 (0)
> - _A_ src/assets/img/p4-header.jpg
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-5 (0)
> - _A_ src/assets/img/p5-header.jpg
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-6 (0)
> - _R_ src/assets/img/p6-header.jpg
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-7 (0)
> - _R_ src/assets/img/p7-header.jpg
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-8 (0)
> - _M_ src/pages/principle1.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-9 (4)
> - _M_ src/pages/principle2.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-10 (11)
> - _M_ src/pages/principle3.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-11 (93)
> - _A_ src/pages/principle4.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-12 (56)
> - _A_ src/pages/principle5.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-13 (50)
> - _A_ src/pages/principle6.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-14 (66)
> - _A_ src/pages/principle7.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-15 (62)
> - _M_ src/partials/nav-simple.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-16 (8)
> - _M_ src/partials/p1/p1-1.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-17 (13)
> - _M_ src/partials/p1/p1-2.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-18 (4)
> - _M_ src/partials/p1/p1-3.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-19 (5)
> - _M_ src/partials/p1/p1-4.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-20 (6)
> - _M_ src/partials/p1/p1-5.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-21 (4)
> - _A_ src/partials/p3/p3-1.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-22 (17)
> - _A_ src/partials/p3/p3-2.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-23 (10)
> - _A_ src/partials/p3/p3-3.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-24 (0)
> - _A_ src/partials/p3/p3-4.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-25 (0)
> - _A_ src/partials/p3/p3-5.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-26 (0)
> - _A_ src/partials/p4/p4-1.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-27 (0)
> - _A_ src/partials/p4/p4-2.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-28 (0)
> - _A_ src/partials/p4/p4-3.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-29 (0)
> - _A_ src/partials/p4/p4-4.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-30 (1)
> - _A_ src/partials/p5/p5-1.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-31 (0)
> - _A_ src/partials/p5/p5-2.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-32 (0)
> - _A_ src/partials/p5/p5-3.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-33 (0)
> - _A_ src/partials/p6/p6-1.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-34 (0)
> - _A_ src/partials/p6/p6-2.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-35 (0)
> - _A_ src/partials/p6/p6-3.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-36 (0)
> - _A_ src/partials/p6/p6-4.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-37 (0)
> - _A_ src/partials/p7/p7-1.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-38 (0)
> - _A_ src/partials/p7/p7-2.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-39 (0)
> - _A_ src/partials/p7/p7-3.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-40 (0)
> - _A_ src/partials/p7/p7-4.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-41 (0)
> - _A_ src/partials/p7/p7-5.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-42 (0)
> - _A_ src/partials/p7/p7-6.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-43 (0)
> - _A_ src/partials/p7/p7-7.html
>   https://github.com/redaikidoka/ebfm/pull/3/files#diff-44 (0)
> 
> Patch Links:
> - https://github.com/redaikidoka/ebfm/pull/3.patch
> - https://github.com/redaikidoka/ebfm/pull/3.diff
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/redaikidoka/ebfm/pull/3, or mute the thread
> https://github.com/notifications/unsubscribe-auth/AEnlQcNPf1FCWrbVfLJbqx105YYy9tKhks5qbx1FgaJpZM4Jad9v
> .

stancheta committed:
Completed Half of P2, All of P3

New Features:

Added CSS Class that allows for vertical height match
Added Youtube embedded video
Added CSS Class that allows for responsive embedded video
